src/WebCrawler.Shared.DevOps/CrawlerBootstrapper.cs

```diff
@@ -197,6 +202,44 @@ public static AkkaConfigurationBuilder WithOps(
             }
             #endregion
 
+            #region Azure discovery setup
+
+            if (options.StartupMethod is StartupMethod.AzureDiscovery)
+            {
+                Console.WriteLine("From environment: Forming cluster using Akka.Discovery.Azure");
+                
+                builder.WithAkkaManagement(setup =>
+                {
+                    setup.Http.HostName = options.Ip ?? "localhost";
+                    setup.Http.Port = managementPort;
+                    setup.Http.BindHostName = "0.0.0.0";
+                    setup.Http.BindPort = managementPort;
+                });
+                
+                // Add Akka.Management.Cluster.Bootstrap support
+                var connectionString = config.GetConnectionString("azure-tables");
+                builder
+                    .WithClusterBootstrap(setup =>
+                    {
+                        setup.ContactPointDiscovery.RequiredContactPointsNr = 3;
+                        setup.ContactPointDiscovery.StableMargin = TimeSpan.FromSeconds(5);
+                        setup.ContactPointDiscovery.ContactWithAllContactPoints = true;
+                        setup.ContactPointDiscovery.ServiceName = options.Discovery.ServiceName;
+                        setup.ContactPoint.FilterOnFallbackPort = false;
```

Comment:
This is a very important undocumented option. Normally, Akka management port is a fixed port for all nodes with an optional fallback port; if akka failed to find a specific port or port name, it will filter the port list returned by discovery by the fallback port. This means that a node can have at maximum 2 listed management port, anything else are discarded.

`FilterOnFallbackPort` option allows users to override this, allowing discovery to return multiple ports, even from a single machine. This allows all machines in the cluster to have differing management port, which will be the case when running multiple Akka cluster node in a single machine for development.